integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/json/ESBJAVA4781EscapeAutoPrimitiveTestCase.java

```diff
@@ -54,7 +53,6 @@ public void testJSONEmptyArrayMissingNHTTPTransport() throws Exception {
 
     @AfterClass(alwaysRun = true)
     public void destroy() throws Exception {
-        super.cleanup();
```

Comment:
shall we remove method  itself. lets update other places also

Reply:
done.